wgsl/index.bs

```diff
@@ -3408,11 +3408,14 @@ compound_statement
 ## Assignment TODO ## {#assignment}
 
 <pre class='def'>
-assignment_statement
-  : singular_expression EQUAL short_circuit_or_expression
-      If singular_expression is a variable, this maps to OpStore to the variable.
+assignable_expression
+  : IDENT (postfix_expression)*
+    If assignable_expression is a variable, this maps to OpStore to the variable.
       Otherwise, singular expression is a pointer expression in an Assigning (L-value) context
       which maps to OpAccessChain followed by OpStore
+
+assignment_statement
```

Comment:
There is already an assignment_statement production

Does this PR mean to add to it, or replace it?

Reply:
Github diff showing it in a weird way, but hey, it's just 6 lines!
The PR only changes the `assignment_statement`, doesn't add a new one

Reply:
Ah, I see that.
I was tripped up  by the fact that you've replaced singular_expression with `IDENT (postfix_expression)*`

That's a regression in the grammar that prevents implementation of the ptr/ref resolutions from #1456 and #1530.